cli/cmd/debug/bpf/bpf_linux.go

```diff
@@ -0,0 +1,13 @@
+package bpf
+
+import "github.com/spf13/cobra"
+
+var Cmd = &cobra.Command{
+	Use:   "bpf",
+	Short: "BPF debug commands",
+}
+
+func init() {
+	Cmd.AddCommand(featuresCmd)
+	Cmd.AddCommand(qdiscCmd)
+}
```

Comment:
Let's add a `status` command, showcasing the important one-liners:
- socket setup
- agent running (maybe)
- maps
- programs
- flow rate (?)
- plugins

Reply:
you can already get flow rate by exec `hubble status` on the agent so it might be overkill?